app/app.go

```diff
@@ -7,12 +7,19 @@ import (
 	"strings"
 )
 
-// extraArgs contains extra arguments to append to
-// os.Args. The arguments are separated with |.
-// Useful for running programs on mobiles where the
-// command line is not available.
-// Set with the go linker flag -X.
-var extraArgs string
+var (
+	// extraArgs contains extra arguments to append to
+	// os.Args. The arguments are separated with |.
+	// Useful for running programs on mobiles where the
+	// command line is not available.
+	// Set with the go linker flag -X.
+	extraArgs string
+	// appID specifies the package name for Android and Wayland
+	// or the bundle id for iOS and tvOS.
+	// appID can be set by using either AppID configuration option
+	// or using gio-cmd -appid flag.
```

Comment:
We don't actually have a `-target wayland` yet, right?

Reply:
Perhaps it should be `-target linux` anyway?

Reply:
Changed that comment